docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -66,5 +63,18 @@ To check if you have this driver installed:
 4. Click the **Driver** tab to see the driver version. 
    ![](../img/DeviceDriverVersion.PNG)
 
-You are done updating your device driver and are ready to use your GPU. Proceed to the <a href="openvino_docs_install_guides_installing_openvino_windows.html#get-started">Start Using the Toolkit</a> section to learn the basic OpenVINO™ toolkit workflow and run code samples and demo applications.
+
```

Comment:
You are done updating your device driver and are ready to use your GPU.

Reply:
```suggestion
You are done updating your device driver and are ready to use your GPU.
```